views/index.html

```diff
@@ -1,172 +0,0 @@
-            </ul>
-          </div>
-          <a href="http://www.mozilla.org/" id="tabzilla">mozilla</a>
-        </nav>
-      </header>
-      {% endblock %}
-
-      <script src="{{ personaHost }}/include.js"></script>
-      <script src="/bower/webmaker-login-ux/dist/webmakerLogin.js"></script>
-    </div>
-
-    <div class="ui-wrapper ui-body-nobg">
-      <div class="banner">
-        <div class="header">
-          <div>
-            <img class="logo" src="img/goggles.png" alt="{{ gettext("X-Ray Goggles Logo") }}">
-          </div>
-          <div>
-            <h1>{{ gettext("X-Ray Goggles") }}</h1>
-            <h2>{{ gettext("Remix and share web pages instantly") }}</h2>
-          </div>
-        </div>
-        <div class="bookmarklet"><a href="" id="bookmarklet-link" class="ui-btn"><span>{{ gettext("Activate X-Ray Goggles") }}</span></a></div>
-      </div>
-      <div class="ui-bg-column">
-        <div class="ui-row">
-          <div class="g-4">
-            <h1>{{ gettext("See how Goggles work by swapping an image") }}</h1>
-            <ol>
-              <li>
-                <p>{{ gettext("Copy this image URL") }}<br>
-                <input type="text" autofocus value="{{ host }}/img/goggles-remixed.png"></p>
-                <p>{{ gettext("The URL you just copied links to a new image!") }}</p>
-              </li>
-              <li>
-                <p>{{ gettext("activate the X-Ray Goggles by") }}</p>
-              </li>
-              <li>
-                <p>{{ gettext("With X-Ray Goggles now activated") }}</p>
-              </li>
-              <li>
-                <p>{{ gettext("Find the current X-Ray Goggles image") }}</p>
-                <p><img class="example" src="img/goggles.png" alt="{{ gettext("X-Ray Goggles Logo") }}"></p>
-                <p>{{ gettext("Then click on the highlighted image") }}</p>
-              </li>
-              <li>
-                <p>{{ gettext("Replace the image location") | safe }}</p>
-                <p><strong>{{ gettext("Here's how:") }}</strong> {{ gettext("See where it says") | safe }}</p>
-              </li>
-              <li>
-                <p>{{ gettext("Great! You've successfully swapped images") }}</p>
-              </li>
-            </ol>
-          </div>
-
-          <div class="g-4">
-            <h1>{{ gettext("Share your remix") }}</h1>
-            <p>{{ gettext("When you're ready to share your remixed page") | safe }}</p>
-
-            <h1>{{ gettext("Help") }}</h1>
-            <p>{{ gettext("If you need help") | safe }}</p>
-
-            <h1>{{ gettext("Remix any webpage!") }}</h1>
-            <p>{{ gettext("You can take X-Ray Goggles with you anywhere on the web:") }}</p>
-            <ol id="install">
-              <li>
-                <p>{{ gettext("Make sure your web browser's bookmarks bar is enabled.") }} </p>
-                <p class="screenshot step1" data-step="1"><img src="img/instructions/firefox1.png"></p>
```

Comment:
Actually, `/public/js/browser-screen.js` uses them. I'll leave them there for now.